templates/component/telemetryPrompt.html.twig

```diff
@@ -0,0 +1,16 @@
+<div class="alert alert-info">
+    <h4>Telemetry</h4>
+    <p>
+        Help us improve <strong>Repman</strong> by enabling sending anonymous usage statistic.
+    </p>
+    <div class="btn-list">
+        <form class="card" action="{{ path('admin_config_telemetry_enable') }}" method="POST">
+            <input type="hidden" name="_method" value="POST">
+            <button class="btn btn-info" type="submit">Okay</button>
+        </form>
+        <form class="card" action="{{ path('admin_config_telemetry_disable') }}" method="POST">
+            <input type="hidden" name="_method" value="DELETE">
+            <button class="btn btn-secondary" type="submit">No, thanks</button>
+        </form>
+    </div>
+</div>
```

Comment:
<img width="723" alt="Screenshot 2020-07-17 at 18 06 44" src="https://user-images.githubusercontent.com/3144291/87807271-5b243c00-c858-11ea-973b-e89e332103ba.png">

Reply:
this should only be visible to the instance admin

Reply:
ok, now i see :facepalm: